upd_script/fetch_sensors.sh

```diff
@@ -25,8 +25,12 @@ else
     # change_branch $BRANCH  # change to a branch we're working on, if we've defined the branch above.
 fi
 
-sudo python $SENSOR_DIR/Python/setup.py install
-sudo python3 $SENSOR_DIR/Python/setup.py install
+cd $SENSOR_DIR/Python
```

Comment:
can you use pushd/popd instead? Otherwise you won't be in your starting folder when this is done.